tests/test_api/test_validation.py

```diff
@@ -230,6 +370,10 @@ def test_validate_jobs():
     granule_with_dem_coverage = 'S1A_IW_SLC__1SSV_20150621T120220_20150621T120232_006471_008934_72D8'
     granule_without_dem_coverage = 'S1A_IW_GRDH_1SDV_20201219T222530_20201219T222555_035761_042F72_8378'
 
+    valid_ref_burst = 'S1_136231_IW2_20200604T022312_VV_7C85-BURST'
+    valid_sec_burst = 'S1_136231_IW2_20200616T022313_VV_5D11-BURST'
+    invalid_sec_burst = 'S1_136232_IW2_20200616T022313_VV_5D11-BURST'
```

Comment:
I forgot how `test_validate_jobs` works; what specifically makes `invalid_sec_burst` an invalid granule?

Reply:
It's not valid because it's id doesn't match the above granules id. I've made this more clear by creating a valid_burst_pair and invalid_burst_pair.